cs-engine/1.13/index.md

```diff
@@ -162,30 +223,100 @@ are available:
+
+### Package download links
+
+{% assign rpm-prefix = "https://packages.docker.com/1.13/yum/repo/main" %}
+{% assign deb-prefix = "https://packages.docker.com/1.13/apt/repo/pool/main/d/docker-engine" %}
+
+#### CS Docker Engine 1.13.1
+
+{% comment %} Check on the S3 bucket for packages.docker.com for the versions. {% endcomment %}
+{% assign rpm-version = "1.13.1.cs1-1" %}
+{% assign rpm-rhel-version = "1.13.1-cs2-1" %}
+{% assign deb-version = "1.13.1~cs2-0" %}
+
+| Operating System      | Binary package Links                                                                                                                                                                                                                                                                                                                                               | Source package Links                                                                                                                                                                                                           |
+|-----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| RHEL 7.x and CentOS 7 | [docker-engine]({{ rpm-prefix }}/centos/7/Packages/docker-engine-{{ rpm-version}}.el7.centos.x86_64.rpm), [docker-engine-debuginfo]({{ rpm-prefix }}/centos/7/Packages/docker-engine-debuginfo-{{ rpm-version }}.el7.centos.x86_64.rpm), [docker-engine-selinux]({{ rpm-prefix }}/centos/7/Packages/docker-engine-selinux-{{ rpm-version}}1.el7.centos.noarch.rpm) | [docker-engine]({{ rpm-prefix }}/centos/7/Packages/docker-engine-{{ rpm-version }}.el7.centos.src.rpm), [docker-engine-selinux]({{ rpm-prefix }}/centos/7/Packages/docker-engine-selinux-{{ rpm-version }}.el7.centos.src.rpm) |
+| RHEL 7.2 (only use if you have problems with `selinux` with the packages above) | [docker-engine]({{ rpm-prefix }}/rhel/7.2/Packages/docker-engine-{{ rpm-rhel-version }}.el7.centos.x86_64.rpm), [docker-engine-debuginfo]({{ rpm-prefix }}/rhel/7.2/Packages/docker-engine-debuginfo-{{ rpm-rhel-version }}.el7.centos.x86_64.rpm), [docker-engine-selinux]({{ rpm-prefix }}/rhel/7.2/Packages/docker-engine-selinux-{{ rpm-rhel-version }}.el7.centos.noarch.rpm) | [docker-engine]({{ rpm-prefix }}/rhel/7.2/Packages/docker-engine-{{ rpm-rhel-version }}.el7.centos.src.rpm), [docker-engine-selinux]({{ rpm-prefix }}/rhel/7.2/Packages/docker-engine-selinux-{{ rpm-rhel-version }}.el7.centos.src.rpm) |
```

Comment:
They aren't broken anymore with the latest commit. Because the lines here didn't have to change (yay templating) Github won't notice.